src/NServiceBus.Persistence.DynamoDB/Saga/SagaPersistenceConfiguration.cs

```diff
@@ -34,5 +35,7 @@ public class SagaPersistenceConfiguration
         /// How long the client should attempt to acquire a lock when using pessimistic locking before giving up.
         /// </summary>
         public TimeSpan LeaseAcquisitionTimeout { get; set; } = TimeSpan.FromSeconds(10);
+
+        internal JsonSerializerOptions MapperOptions { get; set; } = new(DynamoDB.MapperOptions.Defaults);
```

Comment:
No because then you cannot bypass the custom converters anymore. You can only remove converters by instance which would require to expose the converters